db/WiredTiger.turtle

```diff
@@ -3,4 +3,4 @@ WiredTiger 2.8.1: (March 24, 2016)
 WiredTiger version
```

Comment:
Your db folder did not get gitignore'd properly. Just put ``db`` in your gitignore.